src/routes/choice-exercises/components/SingleChoiceListItem.tsx

```diff
@@ -175,7 +175,7 @@ const SingleChoiceListItem = ({
         </StyledArticleText>
       </StyledArticleBox>
       <StyledWord selected={selected} pressed={pressed} correct={showCorrect} delayPassed={delayPassed}>
-        {word}
+        {word} {article.id === 4 ? "(plural)" : " "}
```

Comment:
I think this needs some more work :)
The current way, there is always a space added and for articles different than `die (Plural)` even two spaces are added. There should only be a space if needed though, i.e. before the `(Plural)` text. Can you find a way to do so?

All string constants should be added to the `labels.json` in an appropriate place and used from there.

Also: I think your code style is a little bit off, we always use single quotes `'`. Are you using IntelliJ or something different like VSCode?

Reply:
Hello, thank you for the feedback. For the first point, would "{word}{article.id === 4 && '(Plural)'}" solve the issue ?

Reply:
Yes, I think so. However, I would not put the space into the labels.json. Not sure whether to only put `Plural` or `(Plural)`. Probably even the first one without brackets.

Reply:
In the ticket it was explicitly defines as `(Plural)` with brackets.

Reply:
Yes, but my point was whether to add it to labels.json with brackets or without. The user should always see brackets.

Reply:
ahh, ok I get it. Would put it to labels.json without brackets, so we can reuse it if needed.

Reply:
I agree :)